### PR TITLE
pic changes

### PR DESCRIPTION
**Pull request by cndhart**
pic changes

### CODE REVIEW

File: content/toolrolls.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ menu: "main"
 type: "product"
 weight: 8
 images:
- - "/img/tool1.jpg"
- - "/img/tool2.jpg"
+ - "/img/Std,Met,Shor.jpg"
```
Comment (xaque208) [inline content/toolrolls.md]: Commas in file names aren't great.  Can you replace with underscores please?

Comment (xaque208) [reply content/toolrolls.md]: The files themselves will need to be moved also.

**Comment by cndhart** [comment]
removed from the list of pictures? 
Cindy
cindy@cindycolorado.com

**Comment by cndhart** [comment]
is the FILE mean the name like tool1.jpg? and how do I remove it? Can’t I just type the name over it that I want there? You are so computer savvy and I am trying to understand everything so I do it right.

Mom
Cindy
cindy@cindycolorado.com

**Comment by cndhart** [comment]
Ok I changed that little thing.  Now can you tell if I did everything like it is supposed to done?

Cindy
cindy@cindycolorado.com

**Comment by xaque208** [comment]
Nice, this looks good.

**Comment by xaque208** [comment]
So now that branch that you created has been merged to master.  Now locally, go back to the master branch and `pull`, and you'll see the updated changes.  Then to make more changes, you can create a new branch.  Usually the branch name will reflect the work involved, so something somewhat descriptive so in case you don't come back to the work right away, you remember what it was that you were working on.

**Comment by cndhart** [comment]
Ok, so now why does it cut off part of the pics?

Cindy
cindy@cindycolorado.com

**Comment by xaque208** [comment]
Its cropping as part of the thumbnail.  I can increase the size pretty easy, but I think it won't crop when I fix the image zoom thing.

**Comment by cndhart** [comment]
Oh good enough then.  I’ll just be patient and carry on.  I’m having fun learning this stuff.

Love you!
Cindy
cindy@cindycolorado.com

**Pull request merged**